docs/usage/compact.rst.inc

```diff
@@ -66,11 +66,11 @@ Depending on the amount of segments that need compaction, it may take a while,
 so consider using the ``--progress`` option.
 
 A segment is compacted if the amount of saved space is above the percentage value
-given by the ``--threshold`` option. If ommitted, a threshold of 10% is used.
+given by the ``--threshold`` option. If omitted, a threshold of 10% is used.
 When using ``--verbose``, borg will output an estimate of the freed space.
 
 After upgrading borg (server) to 1.2+, you can use ``borg compact --cleanup-commits``
 to clean up the numerous 17byte commit-only segments that borg 1.1 did not clean up
 due to a bug. It is enough to do that once per repository.
 
-See :ref:`separate_compaction` in Additional Notes for more details.
\ No newline at end of file
+See :ref:`separate_compaction` in Additional Notes for more details.
```

Comment:
this change should be removed in **this** file - it will be regenerated from source at release time.

Reply:
i reverted this change in the autogenerated file.